katana/native/cases/views.py

```diff
@@ -0,0 +1,370 @@
+			details = py_file_details(path_to_pythonsrc);
+	else:
+			details = All_case_action_details;
+			
+	driver  = request.GET.get('driver');
+	keyword = request.GET.get('keyword');
+	responseBack = { 'fields': [] }
+	
+	print "LOOOK::-->",  driver, " ", keyword;
+
+	if driver == "To_Be_Developed" :
+		return JsonResponse(responseBack)
+
+	try:
+		items = details[driver]
+	except:
+		return JsonResponse(responseBack)
+	print "inputs = ", driver,  keyword
+	print "items == ", items;
+	try: 
+		for item in details[driver][0]: 
+			print "fn ==> ", item['fn'], driver, keyword
+			if item['fn'] == keyword: 
+				print item.keys();
+				responseBack['fields'].append(item);
+				break;
+	except:
+		print details[driver]
+	return JsonResponse(responseBack)
+
+def getEmpty():
+	edata={
+		  "Testcase": {
+		    "Details": {
+		      "Name": "set_env_variable",
+		      "Title": "set_env_variable",
+		      "default_onError": { "@action": "next" },
+		      "Date": "2017-01-01",
+		      "Time": "23:00",
+		      "InputDataFile": "No_Data",
+		      "Engineer": "Warrior_Test",
+		      "Category": "Regression",
+		      "State": "Released"
+		    },
+		    "Requirements": {
+		      "Requirement": [
+		        "Demo-requirement-001",
+		        "Demo-requirement-002",
+		        "Demo-requirement-003"
+		      ]
+		    },
+		    "Steps": {
+		      "step": {
+		        "@Driver": "common_driver",
+		        "@Keyword": "set_env_var",
+		        "@TS": "1",
+		        "Arguments": {
+		          "argument": [
+		            {
+		              "@name": "var_key",
+		              "@value": "check3"
+		            },
+		            {
+		              "@name": "var_value",
+		              "@value": "3"
+		            }
+		          ]
+		        },
+		        "Description": [
+		          "Regression for existing support, setting one ENV variable",
+		          "compare values"
+		        ],
+		        "onError": { "@action": "next" },
+		        "Execute": { "@ExecType": "Yes" },
+		        "context": "positive",
+		        "impact": "impact"
+		      }
+		    }
+		  }
+		}  ;
+
+	return edata; 
+
+def editCase(request):
+	""" 
+	Set up JSON object for editing a suites file. 
+	"""
+	path_to_config_file = navigator.get_katana_dir() + os.sep + "config.json"
+	x= json.loads(open(path_to_config_file).read());
+	path_to_testcases = x['xmldir'];
+	template = loader.get_template("editCase.html")
+	filename = request.GET.get('fname')
+
+	# Open the XML file and get it's dictionary...
+	# Make exceptions for missing or badly formatted files. 
+	
+	# Set up defaults for an xml_r object
+
+	print "Asked for ", filename
+	print "Path to cases ", path_to_testcases
+	if filename.find("..") == 0: 
+		f = filename.find('testcases')
+		print "f == ", f, filename 
+		if f > -1: 
+			filename = os.path.dirname(path_to_testcases) + os.sep + filename[f:]
+		else: 
+			filename = path_to_testcases + os.sep + filename
+	print "Attempting to read ...", filename 
+
+	xml_r = {}
```

Comment:
Instead of creating a dict - use an empty xml file (keep that in the static directory) and that would serve as a base template.

Reply:
Done. File is called empty dot xml in the same directory as views.py

Reply:
Can this empty.xml file be moved into the static directory? You can create a directory, say, base_templates and store the file there.